kinesis-produce-consume/src/main/java/kinesis/KinesisApplication.java

```diff
@@ -0,0 +1,18 @@
+package kinesis;
```

Comment:
Looks like all other samples are in the package `demo`.
IMO it's wrong and must be like `org.springframework.cloud.stream.sample.*`, but that is already late.
So, rename it, please, to the `demo` anyway. At least we will be consistent with existing state.

Also be sure to add `Copyright` in the beginning of each Java file.